src/components/atoms/BadgePreview/v2/BadgePreview.stories.tsx

```diff
@@ -19,7 +20,7 @@ const EXAMPLE_URLS = {
 }
 
 export default {
-  title: 'Components/Molecules/BadgePreviewV2',
+  title: 'Components/Atoms/BadgePreview/v2',
```

Comment:
Just to be on the same page, why a BadgePreview would be an `Atom` and not a `Molecule`?

Reply:
But also Atoms are considered not useful on their own, and BadgePreview is a fully functional component on its own its some kind of core component with functionality ... Anyways is some kind of filosofical discussion haha, I'm fine putting it in one or another place, I just wanted to lear how to certificate it properly 👍